### DISCUSSION TITLE
如何接入知识库问答？

### DISCUSSION THREAD

**Discussion by shenlongju11** [zai-org/ChatGLM3#1186] [Q&A] [answered]
除了微调，一个langchain项目使用的chatchat模型可以导入word，excel文件作为知识库。ChatGLM3能否结合知识库？请大仙指点。

**Comment by zRzRzRzRzRzRzR** [comment, marked as answer]
那你用chatchat接入就行了，接入知识库是工程问题，应该考虑使用工程方式解决，比如你提到的langchain
